scripts/create_tables.sql

```diff
@@ -1 +1,24 @@
--- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender CHAR(1),
+  born_on TEXT,
+  specialty TEXT
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender CHAR(1),
+  born_on TEXT,
+  insurance TEXT
+);
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  app_date TEXT,
+  app_time TEXT,
+  comment TEXT
+);
```

Comment:
where is the doctor/patient info on your appointments table?? 

Here's 1 possible solution:
```sql
CREATE TABLE appointments(
  date TEXT, time TEXT, comment TEXT,
  doctor_id INTEGER REFERENCES doctors(id),
  patient_id INTEGER REFERENCES patients(id)
);
```